scripts/BuildJSON/build_json.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+'''
+build_json
+----------------------------------
+builds a .json file used to configure the app
+'''
+
+from os.path import join
+from settings import *
+import pyodbc
+import json
+
+
+json_file = join(JSONDIR, 'config.json')
+connection = pyodbc.connect(CONNECTION_STRING)
+cursor = connection.cursor()
+
+
+def get_data(table, fields):
+    cursor.execute('SELECT {} FROM {}'.format(','.join(fields), table))
+    return [list(row) for row in cursor.fetchall()]
+
+
+def make_lu(table, key, value):
+    cursor.execute('SELECT {},{} FROM {}'.format(key, value, table))
+    obj = {}
+    for row in get_data(table, [key, value]):
+        obj[row[0]] = row[1]
+
+    return obj
+
+
+actions_lu = make_lu('LU_ACTION', 'ActionID', 'ActionDescription')
+treatments_lu = make_lu('LU_TREATMENTTYPE', 'TreatmentTypeID', 'TreatmentTypeDescription')
+obj = {
+    'projectStatus': get_data('LU_STATUS', ['StatusDescription', 'StatusID']),
+    'featureType': get_data('LU_FEATURETYPE', ['FeatureTypeDescription', 'FeatureTypeID']),
+    'featureAttributes': {}
+}
+
+# add actions and treatments to polygon features
+cursor.execute("""
+SELECT FeatureTypeDescription, FeatureTypeID FROM LU_FEATURETYPE
+WHERE FeatureClassAssociation = 'Poly'
+""")
+for row in cursor.fetchall():
```

Comment:
No. I haven't added controls for the herbicide stuff yet. I'll get the data when I do that.